### ISSUE TITLE
New Pageview - Citations

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1182]
# TL;DR
With the CHS merger, we will be losing a list of publications that's shown at https://childrenhelpingscience.com/publications

At a minimum, we should rescue this list to a near-identical page on our site, so that https://childrenhelpingscience.com/publications will continue to resolve to a citation list. 

# Narrative
"As a researcher, or as a funder/potential funder, I would like to see a list of other studies that have been conducted on the platform."

# Implementation notes

This view does *not* need to be linked in the top bar or otherwise added to navigation; it just to exist/be linkable.
